.github/workflows/ca-sequential-test.yml

```diff
@@ -1836,6 +1836,551 @@ jobs:
+                  -D "cn=Directory Manager" \
+                  -w Secret.123 \
+                  -b $DN \
+                  -s base \
+                  -o ldif_wrap=no \
+                  -LLL \
+                  | grep \
+                      -e SecurePort: \
+                      -e beginRange: \
+                      -e endRange: \
+                      -e host: \
+                  | sort >> actual
+
+              echo >> actual
+          done
+
+          # new request range should be 31 - 40 decimal (total: 10)
+          cat > expected << EOF
+          SecurePort: 8443
+          beginRange: 11
+          endRange: 20
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 21
+          endRange: 30
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 31
+          endRange: 40
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 41
+          endRange: 50
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 51
+          endRange: 60
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 61
+          endRange: 70
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 71
+          endRange: 80
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 81
+          endRange: 90
+          host: pki.example.com
+
+          EOF
+
+          diff expected actual
+
+      - name: Check cert range objects
+        run: |
+          docker exec ds ldapsearch \
+              -H ldap://ds.example.com:3389 \
+              -D "cn=Directory Manager" \
+              -w Secret.123 \
+              -b ou=certificateRepository,ou=ranges,dc=ca,dc=pki,dc=example,dc=com \
+              -s one \
+              -o ldif_wrap=no \
+              -LLL | tee output
+
+          rm -f actual
+
+          for DN in $(sed -n 's/^dn: *\(.*\)$/\1/p' output)
+          do
+              docker exec ds ldapsearch \
+                  -H ldap://ds.example.com:3389 \
+                  -D "cn=Directory Manager" \
+                  -w Secret.123 \
+                  -b $DN \
+                  -s base \
+                  -o ldif_wrap=no \
+                  -LLL \
+                  | grep \
+                      -e SecurePort: \
+                      -e beginRange: \
+                      -e endRange: \
+                      -e host: \
+                  | sort >> actual
+
+              echo >> actual
+          done
+
+          # new cert range should be 27 - 42 decimal (total: 16)
+          cat > expected << EOF
+          SecurePort: 8443
+          beginRange: 11
+          endRange: 26
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 27
+          endRange: 42
+          host: pki.example.com
+
+          SecurePort: 8443
+          beginRange: 55
+          endRange: 64
```

Comment:
The range size is now 10 instead of 16.

Reply:
Yes, this is intentional. See comment above. If there is a reason to not modify this I'll change. Let me know

Reply:
The main goal is to fix the gap, but since it might not be possible to fully test the impact of the migration on existing deployments under all possible scenarios, we should also try to minimize the changes including the range size. If the admin wants to change the size they can do that after the migration is done.